grails-app/taglib/org/pih/warehouse/FormatTagLib.groovy

```diff
@@ -120,13 +106,13 @@ class FormatTagLib {
             Locale locale = attrs.containsKey('locale') ? attrs.locale : session?.user?.locale ?: defaultLocale
 
             // default format is to display the localized name of the catergory
-            def value = LocalizationUtil.getLocalizedString(attrs?.category?.name?.encodeAsHTML(), locale)
+            def value = LocalizationUtil.getLocalizedString(attrs?.category?.name, locale)
             if (attrs.shorten) {
                 if (value.length() > 35) {
                     value = value.substring(0, 35) + "..."
                 }
             }
-            out << value
+            out << value.encodeAsHTML()
```

Comment:
Here we call `encodeAsHTML()` on the string _after_ it's been localized, not before